routes/checkspell.js

```diff
@@ -17,13 +18,19 @@ const handleResponse = initialWords => {
 
 /* POST checkspell listing. */
 router.post("/", function(req, res) {
+  let url = config.default_tamil_spellchecker
+  let body_config = config.default_tamil_spellchecker_body
```

Comment:
change this to camelCase.
```suggestion
  let bodyConfig = config.default_tamil_spellchecker_body
```